use_case_guide/real_time_dashboards.rst

```diff
@@ -314,9 +314,41 @@ like this:
 
 .. code-block:: sql
 
+  -- let's first use hll extension on a regular postgres table
+  CREATE TABLE http_request_local AS SELECT * FROM http_request;
```

Comment:
Thanks for opening up this PR!! 

Should we just remove this part here, and focus on the distributed table itself? This is a bit misleading, since it suggests pulling all the data to the master. 

Alternately, we could just clarify this query with a comment saying that these queries are intended to run on the worker shards, and roll-up into an aggregate shard, similar to the examples above.